pcsx2/PAD/Linux/wx_dialog/GamepadConfiguration.cpp

```diff
@@ -135,7 +135,7 @@ void GamepadConfiguration::OnCheckboxChange(wxCommandEvent& event)
 		g_conf.pad_options[m_pad_id].forcefeedback = (m_cb_rumble->GetValue()) ? (u32)1 : (u32)0;
 		if (m_cb_rumble->GetValue())
 		{
-			s_vgamePad[m_pad_id]->TestForce();
+			s_vgamePad[m_joy_map->GetSelection()]->TestForce();
 			m_sl_rumble_intensity->Enable();
 		}
 		else
```

Comment:
Note that below this in OnChoice Change ID is looking at a selection box that doesn't exist. It would be best to update it to use m_joy_map like you've done above so the selection box works as well

Reply:
I'm not sure I understand what you mean. For me the selection box works correctly.
Could you list the steps to get it to break?

Reply:
That was fro a specific adaper and is more of a hack nvm on that one